playbook/app/pb_kits/playbook/pb_legend/docs/example.yml

```diff
@@ -1,11 +1,13 @@
 examples:
-  
+
   rails:
   - legend_default: Default
   - legend_prefix: With Prefix Text
- 
-  
-  
+  - legend_status_colors: With Status Colors
+
+
+
```

Comment:
I understand that this was preexisting, but I'm not sure why all this extra whitespace is here, and I don't think we should keep it.